packages/mantine/src/components/collection/Collection.tsx

```diff
@@ -190,9 +212,17 @@ export const Collection = <T,>(props: CollectionProps<T>) => {
                         {...others}
                     >
                         <Stack spacing={spacing}>
-                            {items}
-                            {provided.placeholder}
-                            {_addButton}
+                            <Stack spacing={'xs'}>
+                                <Title order={3}>{label}</Title>
```

Comment:
to avoid exploding code if you remove the label and description from the default props you can do something like this? (i'm not sur of the syntax by heart 🙈 )

This way also you avoid to add useless divs in the DOM on render 👍 

```suggestion
                           title ??  <Title order={3}>{label}</Title>
```